NEMP/mods.json

```diff
@@ -4206,8 +4206,80 @@
         "function": "CheckForgeJson",
         "active": true,
         "forgejson": {
-            "url": "https://raw.githubusercontent.com/hea3ven/BuildingBricks/master/media/update.json",
-            "mcversion": "1.8.9"
+            "url": "https://raw.githubusercontent.com/hea3ven/BuildingBricks/version/media/update.json",
+            "mcversion": "1.9"
+        }
+    },
+    "BuildingBricks": {
+        "function": "CheckForgeJson",
+        "active": true,
+        "forgejson": {
+            "url": "https://raw.githubusercontent.com/hea3ven/BuildingBricks/version/media/update.json",
+            "mcversion": "1.9.4"
+        }
+    },
+    "HardModeTweaks": {
+        "function": "CheckForgeJson",
+        "active": true,
+        "forgejson": {
+            "url": "https://raw.githubusercontent.com/hea3ven/HardModeTweaks/version/media/update.json",
+            "mcversion": "1.9"
+        }
+    },
+    "HardModeTweaks": {
```

Comment:
You have seperate names, and use `"name" : "HardModeTweaks"` for it to map back to the correct mod name